embed.go

```diff
@@ -2,9 +2,46 @@ package grafana
 
 import (
 	"embed"
+	"io/fs"
+	"path/filepath"
+	"testing/fstest"
+
+	"github.com/grafana/testdata"
+	"github.com/yalue/merged_fs"
 )
 
 // CueSchemaFS embeds all schema-related CUE files in the Grafana project.
 //
 //go:embed cue.mod/module.cue kinds/*.cue kinds/*/*.cue packages/grafana-schema/src/common/*.cue public/app/plugins/*/*/*.cue public/app/plugins/*/*/plugin.json pkg/plugins/*/*.cue
-var CueSchemaFS embed.FS
+var cueSchemaFS embed.FS
+
+var CueSchemaFS fs.FS = merged_fs.MergeMultiple(cueSchemaFS, prefixFS("public/plugins/testdata", testdata.CueSchemaFS))
```

Comment:
Go ember doesn't allow/can't embed files from other go modules so this is a very ugly hack! :cry: